ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -207,6 +207,8 @@ def _permute_dims_helper(draw):
 
 
 # permute_dims
+
```

Comment:
There shouldn't be spaces between the comment and the function, same with the other functions in this file